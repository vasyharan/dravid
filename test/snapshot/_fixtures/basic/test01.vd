def foo(a, b, c) {
   val bar = (a + 10) * b - 1 / c
   bar
}

def main() {
  foo(1, 2, 3)
}