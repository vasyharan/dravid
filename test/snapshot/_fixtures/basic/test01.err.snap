SYN: Unexpected (id foo1 14:2)
Expected binary operator
SYN: Unexpected (id foo2 15:2)
Expected binary operator
